releng/org.eclipse.xtext.dev-bom/pom.xml

```diff
@@ -0,0 +1,308 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.eclipse.xtext</groupId>
+	<artifactId>xtext-dev.bom</artifactId>
+	<version>2.17.0-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<description>Xtext Third Party Dependencies (Bill of Materials)</description>
+
+	<properties>
+		<!-- ======== -->
+		<!-- Versions -->
+		<!-- ======== -->
+		<!-- 3rd party -->
+		<antlr-runtime-version>3.2</antlr-runtime-version>
+		<asm-version>7.0</asm-version>
+		<guava-version>[14.0,22.0]</guava-version>
+		<guice-version>3.0</guice-version>
+		<icu4j-version>52.1</icu4j-version>
+		<junit-version>4.12</junit-version>
+		<junit5-version>5.1.0</junit5-version>
+		<log4j-version>1.2.17</log4j-version>
+		<!-- // When adjusting the `lsp4j` version here, do not forget to do the 
+			followings: // - Update the version in `IdeProjectDescriptor#pom`. // - Update 
+			the test expectations by running `CliWizardIntegrationTest#main`. // - Update 
+			the versions in the `/org.eclipse.xtext.ide/META-INF/MANIFEST.MF` and `/org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF` 
+			files. -->
+			
+		<!-- Eclipse Platform -->
+		<!-- Variable naming scheme: -->
+		<!-- * skip 'org.eclipse' prefix, use hyphen separator instead of . -->
```

Comment:
Not sure I like this. If we'd use org.eclipse.foobar-version, searching for the bundle name will yield the version as a grep-result, too.

Reply:
agreed, that would be better. will change.

Reply:
took that because it was already started that way